examples/test/el3702Toel4732/moveDataInEpics.db

```diff
@@ -0,0 +1,31 @@
+# EL3702
+record(waveform,"$(P)ec$(MASTER_ID)-s$(SLAVE_AI)-EL3702_s$(NELM)-AI1-Array"){
+  field(DESC, "$(HWTYPE): AI1: Value Array [$(NELM)]")
+  field(PINI, "$(PINI=1)")
+  field(DTYP, "$(DTYP=asynInt16ArrayIn)")
+  field(INP,  "@asyn($(PORT),$(ADDR=0),$(TIMEOUT=1))T_SMP_MS=$(T_SMP_MS=1)/TYPE=asynInt16ArrayIn/ec$(MASTER_ID).s$(SLAVE_AI).mm.CH1_ARRAY?")
+  field(FTVL, "$(FTVL=SHORT)")
+  field(NELM, "$(NELM)")
+  field(SCAN, "I/O Intr")
+  field(TSE,  "$(TSE=-2)")
+  field(FLNK, "$(P)MoveData.PROC")
+}
+
+# EL4732
+record(waveform,"$(P)ec$(MASTER_ID)-s$(SLAVE_AO)-EL4732_s$(NELM)-AO1-Array"){
+  field(DESC, "$(HWTYPE): AO1: Value Array [$(NELM)]")
+  field(PINI, "$(PINI=1)")
+  field(DTYP, "$(DTYP=asynInt16ArrayOut)")
+  field(INP,  "@asyn($(PORT),$(ADDR=0),$(TIMEOUT=1))T_SMP_MS=$(T_SMP_MS=1)/TYPE=asynInt16ArrayOut/ec$(MASTER_ID).s$(SLAVE_AO).mm.CH1_ARRAY=")
+  field(FTVL, "$(FTVL=SHORT)")
+  field(NELM, "$(NELM)")  
+}
+
+record(acalcout,"$(P)MoveData"){
+  field(DESC, "Move data from EL3702 to EL4732")  
+  field(INAA, "$(P)ec$(MASTER_ID)-s$(SLAVE_AI)-EL3702_s$(NELM)-AI1-Array.VAL")  
+  field(CALC, "AA")
+  field(OUT,  "$(P)ec$(MASTER_ID)-s$(SLAVE_AO)-EL4732_s$(NELM)-AO1-Array.VAL")
```

Comment:
Isn't it sufficient to append PP to the OUT-field of the record to trigger processing.
It would make the FLNK ovsolete.